core/src/main/java/org/geysermc/geyser/configuration/GeyserConfig.java

```diff
@@ -48,8 +48,10 @@
 
 @ConfigSerializable
 public interface GeyserConfig {
+    @Comment("This section contains config options related to Geyser's Bedrock listener.")
```

Comment:
The full sentence seems weird in a config context. If we want to add a description, I would do something like "Settings pertaining to Bedrock connections" or something. Same idea with the below section.

Reply:
https://github.com/GeyserMC/Geyser/pull/5390/commits/d40f292518298c387ea4bb95b956e718a3802228
How's this?